x_and_o/x_and_o.c

```diff
@@ -0,0 +1,163 @@
+#include <stdio.h>
+#include <errno.h>
+#include "x_and_o.h"
+
+void initial_board(char board[BOARD_SIZE][BOARD_SIZE])
```

Comment:
A better implementation of this function will be in one single line:
memset(board, '-', sizeof(board)